app/src/main/java/ca/llamabagel/transpo/ui/search/SearchActivity.kt

```diff
@@ -51,7 +55,21 @@ class SearchActivity : AppCompatActivity() {
         }
 
         viewModel.searchResults.observe(this, Observer {
-            recycler.adapter = SearchAdapter(it)
+            recycler.adapter = SearchAdapter(it, this)
         })
     }
+
+    override fun onItemClicked(item: SearchResult) {
+        when (item) {
+            is SearchResult.StopItem -> {
+                startActivity<TripsActivity>(this) {
```

Comment:
Is this temporary? Item selection should pass the result back to the calling activity.